netket/stats/mpi_stats.py

```diff
@@ -26,28 +26,28 @@ def subtract_mean(x, axis=None):
     return x
 
 
-def mean(a, axis=None):
+def mean(a, axis=None, keepdims=False):
     """
     Compute the arithmetic mean along the specified axis and over MPI processes.
 
     Returns the average of the array elements. The average is taken over the flattened array by default,
     otherwise over the specified axis. float64 intermediate and return values are used for integer inputs.
     """
     # asarray is necessary for the axis=None case to work, as the MPI call requires a NumPy array
```

Comment:
I've just noticed: This comment mentioning `asarray` appears to be obsolete now (since there is no `asarray` here, in contrast to `sum` below). If this indeed works this way also for `axis=None`, now seems like a good occasion to remove the comment. (Or add `asarray` back if it turns out to be still necessary.)
```suggestion
```